encoders/numeric/RandomGaussianEncoder/tests/test_randomgaussianencoderload.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import pickle
+
+from .. import RandomGaussianEncoder
+from .. import TransformEncoder
+
+
+class RandomGaussianEncoderLoadTestCase:
+    def test_randomgaussinencoderload(self):
```

Comment:
better merge these two test cases together and remove `class`

Reply:
And please make sure something is actually tested, tests under the old test were actually being done by `NumericTestCase` base class